More playbar cleanup

bcasselsND opened the pull request:
Set initial elevation range to [-1.00, 2.00]; use strings to force trailing zeros.
Convert min and max values to numbers, before using them.
Remove initial input values from HTML, since they are now set by code.
Fix a bug where range was being disabled.

JKlimovND commented:
When I initially filter by elevation, the entire point cloud seems to have the same color. Then, if I change the elevation window at all (even if I delete a trailing zero), the point cloud updates to show the elevation gradient. Other than that, everything looks good to me.

bcasselsND commented:
Because I changed over to using strings for min and max values, some `+` operations did string concat. That didn't work well. I added conversions to numbers, and now the elevation stuff seems to work better.
